src/tests/unit/tests/background/stores/assessment-store.test.ts

```diff
@@ -2083,40 +2098,46 @@ describe('AssessmentStore', () => {
         await storeTester.testListenerToBeCalledOnce(initialState, finalState);
     });
 
-    it.each([true, false])('onUpdateDetailsViewId', async includeStartData => {
-        const payload: OnDetailsViewInitializedPayload = {
-            detailsViewId: 'testId',
-        } as OnDetailsViewInitializedPayload;
-        const tabId = 1000;
-        const url = 'url';
-        const title = 'title';
-
-        const initialDataBuilder = new AssessmentsStoreDataBuilder(
-            assessmentsProvider,
-            assessmentDataConverterMock.object,
-        );
-        if (includeStartData) {
-            initialDataBuilder.withTargetTab(tabId, url, title, 'initialId');
-        } else {
-            initialDataBuilder.withTargetTab(undefined, undefined, undefined, 'initialId');
-        }
-        const initialState = initialDataBuilder.build();
-
-        const finalDataBuilder = new AssessmentsStoreDataBuilder(
-            assessmentsProvider,
-            assessmentDataConverterMock.object,
-        );
-        if (includeStartData) {
-            finalDataBuilder.withTargetTab(tabId, url, title, payload.detailsViewId);
-        } else {
-            finalDataBuilder.withTargetTab(undefined, undefined, undefined, payload.detailsViewId);
-        }
-        const finalState = finalDataBuilder.build();
+    it.each([true, false])(
+        'onUpdateDetailsViewId with includeStartData=%s',
```

Comment:
Adds coverage for line 451 (previously, this test set all fields of persistedTabInfo to undefined, not persistedTabInfo itself)